lib/pry/repl.rb

```diff
@@ -140,12 +140,6 @@ def handle_read_errors
         should_retry = false
         retry
 
-      # Handle <Ctrl+C> like Bash: empty the current input buffer, but don't
-      # quit.  This is only for MRI 1.9; other versions of Ruby don't let you
-      # send Interrupt from within Readline.
-      rescue Interrupt
-        return :control_c
```

Comment:
I don't think this code is only for 1.9. I'd rather remove the comment stating that this is 1.9 only.

Because when I try this branch and do <Ctrl+C> it raises an error. 

```
Error: (main)>
/Users/andrehjr/work/pry/lib/pry/repl.rb:192:in `readline'
/Users/andrehjr/work/pry/lib/pry/repl.rb:192:in `block in input_readline'
/Users/andrehjr/work/pry/lib/pry/input_lock.rb:114:in `interruptible_region'
/Users/andrehjr/work/pry/lib/pry/repl.rb:191:in `input_readline'
/Users/andrehjr/work/pry/lib/pry/repl.rb:179:in `block in read_line'
/Users/andrehjr/work/pry/lib/pry/repl.rb:132:in `handle_read_errors'
/Users/andrehjr/work/pry/lib/pry/repl.rb:165:in `read_line'
```

Reply:
PR updated, thanks for flagging